php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
```

Comment:
Typo inside comment in word `ownner`. Possible corrections: owner (76), owned (1), owners (0),
Typo inside comment in word `witout`. Possible corrections: without (85), inout (0), witout (0),